### PR TITLE
docs: fixing cards links

### PR DESCRIPTION
**Pull request by DeaMariaLeon**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ dimensions.
 <div class="grid" markdown>
 
   ![Sparse](./assets/images/conference-room-icon.png){width=10%, align=left}
-  <a href="Introduction" style: class="card">Introduction </a>
+  <a href="Introduction/" style: class="card">Introduction </a>
```
Comment (hameerabbasi) [inline docs/index.md:21]:
```suggestion
  <a href="introduction/" style: class="card">Introduction </a>
```

File: docs/index.md [hunk 2]
```diff
@@ -18,23 +18,23 @@ dimensions.
...
   { .card }
 
   ![Sparse](./assets/images/install-software-download-icon.png){width=10%, align=left}
-  <a href="Install" style: class="card">Install</a>
+  <a href="Install/" style: class="card">Install</a>
```
Comment (hameerabbasi) [inline docs/index.md:25]:
```suggestion
  <a href="install/" style: class="card">Install</a>
```

**Review by hameerabbasi — CHANGES_REQUESTED** [review]
This should do it.

**Comment by DeaMariaLeon** [comment]
Thanks!

File: docs/index.md [hunk 3]
```diff
@@ -18,28 +18,27 @@ dimensions.
...
   { .card }
 
   ![Sparse](./assets/images/check-list-icon.png){width=10%, align=left}
-  <a href="construct" class="card">How-to guides</a>
+  <a href="construct/" class="card">How-to guides</a>
   { .card }
 
   ![Sparse](./assets/images/repair-fix-repairing-icon.png){width=10%, align=left}
-  <a href="API/BACKEND" style: class="card">API</a>
+  <a href="api/backend/" style: class="card">API</a>
```
Comment (hameerabbasi) [inline docs/index.md:37]:
```suggestion
  <a href="api/BACKEND/" style: class="card">API</a>
```

Comment (DeaMariaLeon) [reply docs/index.md]: Thank you. For some reason I thought I had fixed this..

**Review by hameerabbasi — APPROVED** [review]
Thanks for the awesome work @DeaMariaLeon!

**Comment by DeaMariaLeon** [comment]
Thank you @hameerabbasi !

**Pull request merged**